src/hooks/useFileManager.tsx

```diff
@@ -384,7 +394,13 @@ export default function useFileManager(
           ))}
         </div>
         <div className="mt-10 flex justify-end">
-          <Cancel onClick={(_) => setArchiveDialogueOpen(null)} />
+          <Button
+            type="button"
+            variant="outline"
+            onClick={(_) => setArchiveDialogueOpen(null)}
```

Comment:
adjust this in all other places too...

```suggestion
            onClick={() => setArchiveDialogueOpen(null)}
```